blockindex/bloomfilterindexer.go

```diff
@@ -297,13 +292,15 @@ func (bfx *bloomfilterIndexer) addLogsToRangeBloomFilter(ctx context.Context, bl
 	}
 }
 
-func (bfx *bloomfilterIndexer) getBloomRangeByIndex(idx uint64) (*bloomRange, error) {
-	bfKey := byteutil.Uint64ToBytesBigEndian(idx)
+func (bfx *bloomfilterIndexer) loadBloomRangeFromDB(br *bloomRange, bfKey []byte) error {
```

Comment:
i am thinking, is it better to use `uint64 index` as input?

Reply:
No. See `err := bfx.loadBloomRangeFromDB(bfx.curRangeBloomfilter, bfx.currRangeBfKey);`

Reply:
i mean pass in index and do the `byteutil.Uint64ToBytesBigEndian` inside